test/pure-docker/servers.yaml

```diff
@@ -0,0 +1,27 @@
+---
+-
+  box: google/gce
+  machine_type: "custom-16-20480"
+  name: pure-docker-test-gcp
+  shell_commands:
```

Comment:
Ideally, we should make the scripts be the same for both except maybe for some "init"

Reply:
is this in reference to the `sed` command when using the virtualbox solution? I agree there is a lot of repeated code here. We can probably run at the majority of this as a shell block in the `Vagrantfile` then any other commands from the yaml afterwards which would be customised to google or vbox specifically?

Reply:
It was mostly due to the repeated code and to avoid the chance of us forgetting to update one of the sections.

I like your either, we have the generic commands on the Vagrant file (or here in another section) and the box specific pre-commands as they are right now.